recipes/dmd/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: dmd
+  version: "2.065.0"
+
+source:
+  fn: dmd.2.065.0.zip
+  url: http://downloads.dlang.org/releases/2014/dmd.2.065.0.zip
```

Comment:
Is it possible to compile these tools and not use the precompiled ones?
Is there already a D compiler in conda?